episodes/multi-dimensional-images.md

```diff
@@ -0,0 +1,449 @@
+---
+title: 'Multi-dimensional images'
+teaching: 30
+exercises: 10
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- How are images with more than 2 dimensions handled in Napari?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Explain how axes are handled in Napari (xyz, channels, time)
+- Open and navigate images with different dimensions in Napari
+- Explain what RGB images are and how they are handled in Napari
+- Split and merge channels in Napari
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Image dimensions / axes
+
+As we saw in the ['What is an image?' episode](./FIXME.md), image pixel values 
+are stored as arrays of numbers with certain dimensions and data type. So far we 
+have focused on grayscale 2D images that are represented by a 2D array:
+
+![](fig/array.png){alt="A diagram comparing the array of numbers and image 
+display for a simplified image of an arrow" width='80%'}
+
+Light microscopy data varies greatly though, and often has more dimensions 
+representing:
+
+- Time (t)
+- Channels (c)
+- Spatial axes (z / y / x)
+
+These images will be stored as arrays with a larger number of dimensions than 2. 
+Let's start with our familiar human mitosis image, and work up to some more 
+complex imaging data.
+
+## 2D
+
+Go to the top menu-bar of Napari and select:  
+`File > Open Sample > napari builtins > Human Mitosis`
+
+![](fig/human-mitosis-napari.png){alt="A screenshot of a 2D image of human cells 
+undergoing mitosis in Napari"}
+
+We can see this image only has two dimensions (or two 'axes' as they're also 
+known) due to the lack of sliders under the image, and by checking its shape in 
+the Napari console:
+
+```python
+image = viewer.layers["human_mitosis"].data
+print(image.shape)
+```
+
+```output
+# (y, x)
+(512, 512)
+```
+
+## 3D
+
+Let's remove the mitosis image and move to a 3D image:  
```

Comment:
Do we need to be clearer on removing the mitosis image first. To get the brain to display properly I first had to delete the selected layer so only the brain image was loaded. 
"remove the mitosis image by clicking the delete layer button at top right of the layer list, then " ....